kafka-client/src/test/java/dev/responsive/kafka/integration/ResponsiveSessionStoreIntegrationTest.java

```diff
@@ -172,6 +170,7 @@ public void shouldComputeSessionAggregate() throws Exception {
     // When:
     final Map<String, Object> properties = getMutablePropertiesWithStringSerdes();
     properties.put(STATESTORE_CACHE_MAX_BYTES_CONFIG, 0);
+    properties.put(ResponsiveConfig.STORE_FLUSH_RECORDS_TRIGGER_CONFIG, 1);
```

Comment:
nit: any reason not to just put these in `getMutableProperites`?

Reply:
Along with the statestore cache config, the stretch goal is that we should run these tests with multiple configurations for each of these two settings. So I wanted to have those two properties split from the rest.